Casks/graalvm-ce-java11.rb

```diff
@@ -0,0 +1,40 @@
+cask 'graalvm-ce-java11' do
+  version '19.3.0'
+  sha256 '5a7eaead66971e25bef2c21d94d0760b54bda13761908545be8c0323df17da4a'
+
+  JVMS_DIR = '/Library/Java/JavaVirtualMachines'.freeze
+  TARGET_DIR = "#{JVMS_DIR}/#{cask}-#{version}".freeze
+
+  # github.com/graalvm/graalvm-ce-builds was verified as official when first introduced to the cask
+  url "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-#{version}/#{cask}-darwin-amd64-#{version}.tar.gz"
+  appcast 'https://github.com/oracle/graal/releases.atom'
+  name 'GraalVM Community Edition (Java 11)'
+  homepage 'https://www.graalvm.org/'
+
+  artifact "#{cask}-#{version}", target: TARGET_DIR
+
+  postflight do
+    # Ensure GraalVM JDK 11 is listed by `/usr/libexec/java_home -V`.
+    macos_dir = "#{TARGET_DIR}/Contents/MacOS"
```

Comment:
This is being fixed in the GraalVM build process so this postflight code should check for the existence of the missing files before creating them.